src/main/java/org/broadinstitute/hellbender/utils/Dirichlet.java

```diff
@@ -0,0 +1,67 @@
+package org.broadinstitute.hellbender.utils;
+
+import org.apache.commons.math3.special.Gamma;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.stream.IntStream;
+
+/**
+ * The Dirichlet distribution is a distribution on multinomial distributions: if pi is a vector of positive multinomial weights
+ * such that sum_i pi[i] = 1, the Dirichlet pdf is P(pi) = [prod_i Gamma(alpha[i]) / Gamma(sum_i alpha[i])] * prod_i pi[i]^(alpha[i] - 1)
+ *
+ * The vector alpha comprises the sufficient statistics for the Dirichlet distribution.
+ *
+ * Since the Dirichlet is the conjugate prior to the multinomial, if one has a Dirichlet prior with concentration alpha
+ * and observes each category i n_i times (assuming categories are drawn from a multinomial distribution pi)
+ * the posterior is alpha_i -> alpha_i + n_i
+ *
+ *
+ * @author David Benjamin &lt;davidben@broadinstitute.org&gt;
+ */
+public class Dirichlet {
+    final double[] alpha;
+
+    public Dirichlet(final double[] alpha) {
+        Utils.nonNull(alpha);
+        Utils.validateArg(alpha.length >= 1, "Dirichlet parameters must have at least one element");
+        this.alpha = Arrays.copyOf(alpha, alpha.length);
```

Comment:
done and done; done; I instead had the two-parameter delegate to the one-parameter constructor, but I think this is in the spirit of your suggestion